assignment-client/src/audio/AvatarAudioStream.h

```diff
@@ -23,7 +23,12 @@ class AvatarAudioStream : public PositionalAudioStream {
     AvatarAudioStream(bool isStereo, int numStaticJitterFrames = -1);
 
 private:
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
+    AvatarAudioStream(const AvatarAudioStream&);
+    AvatarAudioStream& operator= (const AvatarAudioStream&);
+#else
```

Comment:
I suggest adding a new header/source file, something like QtCompatibility.h, and implementing some of the functionality there. That'll result in a smaller and cleaner patch. For instance, something like this:

    #ifndef Q_DISABLE_COPY
        #define Q_DISABLE_COPY(className) \
            className(const className &) = delete;\
            className &operator=(const className &) = delete;
    #endif

And with just that, things get smaller and nicer.